requirements.txt

```diff
@@ -5,6 +5,7 @@ backoff >= 2.2.1
 boto3 >= 1.26.165
 click >= 8.1.7
 colorama >= 0.2.5, < 0.4.7
+cryptography < 43.0.0
```

Comment:
This seems a bit shady. Can we pin this more neatly?

Reply:
Not in this case. We're not using cryptography library directly. It's a dependency on other dependencies. In different Python versions, this could be different versions. We're capping it at before v43.0.0 because that's where the known deprecation error happens. Pegging it strictly could introduce unforeseen side-effects that could result in more bugs. As far as we know, prior to v43.0.0, it was working fine.

See https://github.com/paramiko/paramiko/issues/2419